source/ice_zbgc_shared.F90

```diff
@@ -126,6 +126,7 @@ module ice_zbgc_shared
          thinS     = 0.05_dbl_kind     ! minimum ice thickness for brine
 
       real (kind=dbl_kind), public :: & 
+         dz_ocean1  = 2.03035_dbl_kind,  & ! first ocean layer thickness (m)
```

Comment:
Doesn't match `dz_ocean1=2.3035` defined in `ice_zbgc.F90`.
Also doesn't match 2.30349969863892 in 
`/g/data/ik11/inputs/access-om2/input_20201022/mom_1deg/ocean_vgrid.nc`

It would be better for `dz_ocean1` to be defined in one place, and ideally to be defined automatically from the value used in MOM, rather than being hard-coded or a namelist parameter.

Reply:
I totally agree that dz_ocean1 should be derived from MOM, but I am not sure how. The only way I can think of is to add as a coupling variable to parse from MOM to CICE, but this is not efficient, because it is just a constant (at least in z-coordinate). Any idea?

Thanks for catching the error on the default value (2.03035 is a mistake).

Reply:
I don't know how to implement it, but it would be good to have since different configs have different values, e.g. it's ~1.1m in 0.1deg. Maybe payu could set it as a namelist parameter?

Reply:
oh, I already have it as a namelist parameter for cice_input.nml (L153):

https://github.com/hakaseh/1deg_jra55_iaf/blob/iamip2_his/ice/cice_in.nml

Since this is a ice-BGC specific parameter, I put it under zbgc_nml.